workflow/rocoto_funcs/jedivar.py

```diff
@@ -47,11 +48,23 @@ def jedivar(xmlFile, expdir, do_spinup=False):
     ens_dep = ""
     if HYB_WGT_ENS != "0" and HYB_WGT_ENS != "0.0" and HYB_ENS_TYPE == "1":  # rrfsens
         RUN = 'rrfs'
+        ens_depm1 = ""
+        ens_depm2 = ""
+        ens_depm3 = ""
+        for i in range(1, int(ens_size) + 1):
+            ensindexstr = f'mem{i:03d}'
+            ens_depm1 = ens_depm1 + f'\n     <datadep age="00:05:00"><cyclestr offset="-1:00:00">&COMROOT;/{NET}/{VERSION}/{RUN}.@Y@m@d/@H/fcst/enkf/</cyclestr>{ensindexstr}/<cyclestr>mpasout.@Y-@m-@d_@H.@M.@S.nc</cyclestr></datadep>'
+            ens_depm2 = ens_depm2 + f'\n     <datadep age="00:05:00"><cyclestr offset="-2:00:00">&COMROOT;/{NET}/{VERSION}/{RUN}.@Y@m@d/@H/fcst/enkf/</cyclestr>{ensindexstr}/<cyclestr>mpasout.@Y-@m-@d_@H.@M.@S.nc</cyclestr></datadep>'
+            ens_depm3 = ens_depm3 + f'\n     <datadep age="00:05:00"><cyclestr offset="-3:00:00">&COMROOT;/{NET}/{VERSION}/{RUN}.@Y@m@d/@H/fcst/enkf/</cyclestr>{ensindexstr}/<cyclestr>mpasout.@Y-@m-@d_@H.@M.@S.nc</cyclestr></datadep>'
```

Comment:
suggest adding one or two more spaces before `<datadep>` so that we have a little bit indentation from `<and>` `</and>`